src/style.css

```diff
@@ -0,0 +1,123 @@
+:root {
+  --primary-color: #3d4e61;
+  --secondery-color: #e1e7ed;
+  --shadow-color: #3d4e611a;
+  --background-color: #fff;
+  --icon-color: #687c94;
+  --box-shadow-color-one: #00000040;
+  --box-shadow-color-two: #3d4e6133;
+}
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("fonts/Avenir-Book.ttf")
+    format("truetype");
+  font-weight: 300;
+  letter-spacing: 0;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  position: absolute;
+  margin-top: 20px;
+  width: 100%;
+}
+
+.search-bar__big {
+  display: flex;
+  align-items: center;
+  position: relative;
+  width: calc(100% - 18px);
+  height: 68px;
+  border: 1px solid var(--secondery-color);
+  box-shadow: 0 1px 8px 0 var(--shadow-color);
+  border-radius: 3px;
+  background-color: var(--background-color);
+  color: var(----primary-color);
+}
+
+.search-bar__small {
```

Comment:
just create one base class - .search-bar and then create another one where you will override only needed styles, because now a lot of them are duplicated in this two